src/main/java/it/gov/pagopa/pu/debtpositions/mapper/workflow/DebtPositionRequestMapper.java

```diff
@@ -0,0 +1,10 @@
+package it.gov.pagopa.pu.debtpositions.mapper.workflow;
```

Comment:
move mapper towards workflowhub models into connector.workflow package
use the mapper inside the connector's Service
```suggestion
package it.gov.pagopa.pu.debtpositions.connector.workflow.mapper;
```